spec/integrations/shared/base_render_examples.rb

```diff
@@ -333,6 +333,43 @@ def self.unless_method(_field_name, _object, _options)
                 include_examples 'does not serialize the conditional field'
               end
             end
+
+            context 'enforce_all_conditions is enabled' do
+              let(:local_options) { {x: 1, y: 2, v1: value, v2: other_value} }
+
+              before do
+                Blueprinter.configuration.enforce_all_conditions = true
```

Comment:
Setting and resetting global configurations for testing is not an ideal practice.
Can this be done instead with 
```suggestion
                allow(Blueprinter).to receive(:configuration.enforce_all_conditions).and_return(true)
```
and then, as a bonus, you don't need the `after` block